projects/noloan/admin/app/src/main/proto/Sms.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
Now I see that you copy pasted these protos.
I guess there's no clear way to use the ones from the main app?